packages/core/src/renderer/getStyleSheetForBucket.test.ts

```diff
@@ -93,4 +93,44 @@ describe('getStyleSheetForBucket', () => {
       .filter(Boolean);
     expect(actualMediaQueryOrder).toEqual(mediaQueryOrder);
   });
+
+  it('handles "insertionPoint"', () => {
+    const target = createFakeDocument();
+    const renderer = createDOMRenderer();
+
+    const elementA = target.createElement('style');
+    const elementB = target.createElement('style');
+    const elementC = target.createElement('style');
+
+    elementA.setAttribute('data-test', 'A');
+    elementB.setAttribute('data-test', 'B');
+    elementC.setAttribute('data-test', 'C');
+
+    target.head.appendChild(elementA);
+    target.head.appendChild(elementB);
+    target.head.appendChild(elementC);
+
+    getStyleSheetForBucket('r', target, elementB, renderer);
+    getStyleSheetForBucket('d', target, elementB, renderer);
+
+    expect(target.head.children).toMatchInlineSnapshot(`
+      HTMLCollection [
+        <style
+          data-test="A"
+        />,
+        <style
+          data-test="B"
+        />,
+        <style
+          data-make-styles-bucket="d"
+        />,
+        <style
+          data-make-styles-bucket="r"
+        />,
```

Comment:
Related to my comment in getStyleSheetForBucket.ts, I think this order is wrong? `r` should come before `d` right?

Reply:
@behowell, indeed the order is wrong. Good catch 👍